content/blog/twitter-data-download-explanation.en.md

```diff
@@ -0,0 +1,906 @@
+{
+    "title": "Explaining Twitter's Data Download Feature.",
+    "type": "blog",
+    "slug": "twitter-data-download-explanation",
+    "date": "2019-10-17T16:58:19+02:00",
+    "description": "Twitter provides the option to download the user's personal data from their settings. This post explains the process to get the data as well as the meaning of the data provided.",
+    "featured_image": "/card-icons/Twitter_Black.svg",
+    "tags": [ "twitter", "gdpr", "data", "download", "export" ],
+    "authors": [ "xndr" ]
+}
+
+<img class="offset-image offset-image-left" src="/card-icons/Twitter_Black.svg" style="height: 200px;">
```

Comment:
Thanks for looking for another logo. However, I don't think this one adds any value as users don't know it.

I am currently working on a new 'framework' for featured images anyway, so please just remove the logo for now. I will add an image later.